components/feature/prompts/src/main/res/layout/mozac_feature_prompt_login_prompt.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
This feature won't build because 
```
/builds/worker/checkouts/src/components/feature/prompts/src/main/res/layout/mozac_feature_prompt_login_prompt.xml:11: Error: Possible overdraw: Root element paints background ?android:windowBackground with a theme that also paints a background (inferred theme is @android:style/Theme.Holo) [Overdraw]
[task 2019-12-14T00:07:15.567Z]       android:background="?android:windowBackground"
```

I think we just have to add `tools:ignore="Overdraw"` in the root which I see being used elsewhere?